### DISCUSSION TITLE
Filed to play Live TV with the latest version.

### DISCUSSION THREAD

**Discussion by Chaotic-chaos** [jellyfin/jellyfin-desktop#203] [Q&A] [answered]
Hi, sorry to bother.
I am new to jellyfin & jellyfin-media-player.
I am using the latest version of jellyfin-media-player(windows) and [this](https://hub.docker.com/r/nyanmisaka/jellyfin) jellyfin image.
When using web browser the live TV plays normally, but with jellyfin-media-player it report `media error: loading failed` on the right corner and the server throws error blow:
```shell
jellyfin    | [22:13:14] [ERR] [3] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request: Could not find file '/config/transcodes/770b3739e0124a61a3642f9a27fba9dd.ts'. URL GET /Videos/6fcebd74dbb40208b6cf730bad3d1021/stream.hls.
```

Environment:
- OS: windows, amd64
- jellyfin server version: 10.8.0
- jellyfin host os: linux, amd64
- jellyfin-media-player version: 1.7.0-pre2-x64

Thanks for helping!

**Comment by iwalton3** [comment, marked as answer]
This looks like a server issue. Do you get a similar error in the web player?

**Reply by Chaotic-chaos** [reply]
Nope. I can play with the web player.😥

BTW, thanks for the help~

**Reply by iwalton3** [reply]
Does it work with this version that allows retries when media load failures occur? https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.6.0

I have a suspicion that the web client is actually bugged and relies on the “error” to work.

**Reply by Chaotic-chaos** [reply]
Yes! It works!
Thanks for all the help.
But will the newer version add this feature back? Or  from now I just  cannot update this application....

**Reply by iwalton3** [reply]
This means that the live tv function is relying on a “fallback” path in the application. It also seems like some people using internet radio were having that issue, but breakage with live TV is more concerning.

I guess there are two things:
- First, I’ll make that error case generate a modal which allows triggering the fallback. (Something like “media playback failed, try transcoding?”)
- Second, this needs to get fixed in the server. We should not be relying on an error handler for this.

**Reply by iwalton3** [reply]
I need to make a new prerelease for some unrelated changes anyways, so I’ll include this too.

**Reply by Chaotic-chaos** [reply]
Well, thanks for all these hard work!
Looking forward to the future~

**Reply by iwalton3** [reply]
I have created a pre-release with the change I described. https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.7.0-pre3
